client/src/pages/CreateUserProfile.js

```diff
@@ -0,0 +1,283 @@
+        display: flex;
+        span {
+            font-size: ${theme.typography.size.medium};
+            margin-right: ${theme.typography.size.xxsmall};
+            :first-child {
+                color: ${theme.colors.darkerGray};
+                font-weight: 500;
+            }
+            :last-child {
+                margin: 0;
+            }
+        }
+        &.ant-dropdown-menu-item-active {
+            background-color: ${theme.colors.selago};
+        }
+    }
+`;
+
+const DropdownMenu = ({children}) => {
+    const menu = (
+        <StyledMenu>
+            <Menu.Item>
+               <span>10014</span>
+               <span>13th Ave,</span>
+               <span>New York,</span>
+               <span>NY,</span>
+               <span>USA</span>
+            </Menu.Item>
+            <Menu.Divider/>
+        </StyledMenu>
+    );
+    return <Dropdown overlay={menu}>
+        {children}
+    </Dropdown>
+}
+
+const InputGroup = styled.div`
+    --py: ${theme.typography.size.xxsmall};
+    --my: ${theme.typography.size.xxxlarge};
+
+    color: ${theme.colors.royalBlue};
+    margin: var(--my) 0 var(--my) 0;
+
+    input {
+        background-color: transparent;
+        color: ${theme.colors.darkerGray};
+        font-size: ${theme.typography.size.xlarge};
+        margin-bottom: 3.4rem;
+        padding: var(--py) 0 var(--py) 0;
+        width: 100%;
+    }
+
+    label {
+        font-family: ${theme.typography.font.family.button};
+        font-size: ${theme.typography.size.large};
+        font-weight: 500;
+    }
+
+    .address {
+        input {
+            margin-bottom: 0;
+        }
+        label > p:before {
+            content: ' ';
+            background-image: url(${Marker});
+            background-size: contain;
+            background-repeat: no-repeat;
+            display: inline-block;
+            margin-right: ${theme.typography.size.xxsmall};
+            height: ${theme.typography.size.xlarge};
+            width: ${theme.typography.size.xlarge};
+        }
+
+        label:after {
+            content: 'Enter address, zip code, or city';
+            color: #209d7f;
+            font-size: ${theme.typography.size.xsmall};
+            font-family: ${theme.typography.font.family.body};
+            margin-top: ${theme.typography.size.xsmall};
+        }
+    }
+
+    .underline {
+        color: #209d7f;
+        font-size: ${theme.typography.size.xsmall};
+        font-family: ${theme.typography.font.family.body};
+        margin-top: ${theme.typography.size.xsmall};     
+    }
+`
+
+const UnderlineLink = styled.p`
+    --color: ${theme.colors.darkGray};
+    color: var(--color) !important;
+    a {
+        color: var(--color);
+        text-decoration: underline;
+    }
+    font-size: ${theme.typography.size.small};
+`;
+
+const Submit = styled(SubmitButton)`
+    font-weight: 500;
+`
+const CreateProfile = () => {
+useEffect(() => {
+    let header = document.querySelector('.header');
```

Comment:
why do you need query selectors and using JS to update the styles? 
probably better to avoid this, and use styled components? 
if the changes are for parent components, probably this needs to be configured with the "template" on the route

Reply:
Yes, I agree this is definitely better done with a layout template. It was a dirty way to remove the margin on a parent element and modify the toolbar to appear the same as in the design.